test-jest/cartesian/Brush.spec.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { Brush, LineChart, Line, BarChart } from '../../src';
+import { mockMouseEvent } from '../helper/mockMouseEvent';
+
+describe('<Brush />', () => {
+  const data = [
+    { date: '2023-01-01', value: 10 },
+    { date: '2023-01-02', value: 20 },
+    { date: '2023-01-03', value: 10 },
+    { date: '2023-01-04', value: 30 },
+    { date: '2023-01-05', value: 50 },
+    { date: '2023-01-06', value: 10 },
+    { date: '2023-01-07', value: 30 },
+    { date: '2023-01-08', value: 20 },
+    { date: '2023-01-09', value: 10 },
+    { date: '2023-01-10', value: 70 },
+    { date: '2023-01-11', value: 40 },
+    { date: '2023-01-12', value: 20 },
+    { date: '2023-01-13', value: 10 },
+    { date: '2023-01-14', value: 10 },
+  ];
+
+  // wrap brush with a bar chart to make brush traveler work
+  const barChartWithBrush = (
```

Comment:
Instead of sharing the setup between tests, please move this into the test. That would prevent potential side effects.

Reply:
Why? I think this is fine - especially if the test didn't change any props anyways. The component always gets rendered fresh so there isn't any possible side effects.

Reply:
Well, plainly the object barChartWithBrush could be changed, overwritten, etc. 
One could solve that with Object.freeze() or simply setup per test separately. 

But it is more of a nit, fair enough.